http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Constants.java

```diff
@@ -169,7 +169,14 @@
 
     public static final String ACCEPT_ENCODING = "Accept-Encoding";
 
+    //HTTP server connector creation parameters
     public static final String HTTP_HOST = "Host";
+    public static final String HTTP_PORT = "Port";
+    public static final String HTTP_SCHEMA = "Schema";
+    public static final String HTTP_KEY_STORE_FILE = "keyStoreFile";
```

Comment:
This comes directly from the annotation attribute key in ballerina side, (just used the same string as in annotation attribute key)